src/newbank/server/Customer.java

```diff
@@ -6,14 +6,20 @@
  * The type Customer.
  */
 public class Customer {
-
+    private String userName;
     private ArrayList<Account> accounts;
+    private String password;
+    //private CustomerID id;
```

Comment:
Please consider deleting instead of commenting out.

Reply:
Agree. Removed the line.